### PR TITLE
Issue 1330 documentation changes

### LINKED ISSUE #1330: Search for any instances of "We recommend" in the docs and turn verbiage into a direct recommendation
Often, it is tempting to write, "We recommend..." where you can directly make a recommendation.

For example:

```
We recommend that for best performance, increase the knobber bobber by five clicks.
→                "For best performance, increase the knobber bobber by five clicks.
```

### PR DESCRIPTION
**Pull request by konstantinos-null**
#### What this PR does
Opened this PR to tackle issue 1330 and fix up the related documentation points.
#### Which issue(s) this PR fixes or relates to

Fixes #1330

#### Checklist

- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Comment by 56quarters** [comment]
*Sorry, clicked the wrong button trying to run the workflow*

**Comment by konstantinos-null** [comment]
> _Sorry, clicked the wrong button trying to run the workflow_

Ok! Anything i should be doing on my part?

**Comment by 56quarters** [comment]
> > _Sorry, clicked the wrong button trying to run the workflow_
> 
> Ok! Anything i should be doing on my part?

Nope! Feel free to work on this as you see fit and mark it ready when you'd like review

**Comment by konstantinos-null** [comment]
Hey, I think that most parts that could be altered are done so, the ones remaining were in sentences where if I would change them it would seem too weird I think. 
Still if there are any suggestions I am open!

File: docs/sources/operators-guide/architecture/components/distributor.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ The distributor waits for a successful response from `n`/2 + 1 ingesters, where
 
 ## Load balancing across distributors
 
-We recommend randomly load balancing write requests across distributor instances.
+Randomly load balance write requests across distributor instances.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/architecture/components/distributor.md:104]: Why would a user want to do this? (In either phrasing, it is often helpful to provide rationale.) How about "For better performance, ..." or "For more server stability, ..." (making up suggestions on the spot here)?

File: docs/sources/operators-guide/architecture/components/querier.md [hunk 2]
```diff
@@ -27,8 +27,7 @@ Queriers lazily download the bucket index when they receive the first query for
 
 The bucket index contains a list of blocks and block deletion marks of a tenant. The querier later uses the list of blocks and block deletion marks to locate the set of blocks that need to be queried for the given query.
 
-When the querier runs with the bucket index enabled, the querier startup time and the volume of API calls to object storage are reduced.
-We recommend that you keep the bucket index enabled.
+When the querier runs with the bucket index enabled, the querier startup time and the volume of API calls to object storage are reduced. It is recommended that it is enabled
```
Comment (osg-grafana) [inline docs/sources/operators-guide/architecture/components/querier.md:30]:
```suggestion
When the querier runs with the bucket index enabled, the querier startup time and the volume of API calls to object storage are reduced. To take advantage of these performance improvements and cost reductions, keep the bucket index enabled.
```
@56quarters, is performance being improved and costs being reduced?

File: docs/sources/operators-guide/architecture/components/query-frontend/index.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ The query-frontend internally holds queries in an internal queue.
 In this situation, queriers act as workers that pull jobs from the queue, execute them, and return the results to the query-frontend for aggregation.
 To connect the queriers with the query-frontends, configure queriers with the query-frontend address via the `-querier.frontend-address` flag.
 
-We recommend that you run at least two query-frontend replicas for high-availability reasons.
+Run at least two query-frontend replicas for high-availability reasons.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/architecture/components/query-frontend/index.md:19]:
```suggestion
For high-availability reasons, run at least two query-frontend replicas.
```
Changed the word order to address _why_ before _how_.

File: docs/sources/operators-guide/architecture/components/store-gateway.md [hunk 4]
```diff
@@ -127,8 +127,7 @@ The store-gateway supports the following type of caches:
 - [Chunks cache](#chunks-cache)
 - [Metadata cache](#metadata-cache)
 
-We recommend that you use caching in a production environment.
-For more information about configuring the cache, refer to [production tips]({{< relref "../../run-production-environment/production-tips/index.md#caching" >}}).
+We recommend that you use caching in a production environment. To configure it, refer to [production tips]({{< relref "../../run-production-environment/production-tips/index.md#caching" >}}).
```
Comment (osg-grafana) [inline docs/sources/operators-guide/architecture/components/store-gateway.md:130]:
```suggestion
..., use caching in a production environment. To configure it, refer to [production tips]({{< relref "../../run-production-environment/production-tips/index.md#caching" >}}).
```
Why do we recommend this?

File: docs/sources/operators-guide/configure/configure-otel-collector.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ weight: 150
 
 When using the [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/), you can write metrics into Mimir via two options: `prometheusremotewrite` and `otlphttp`.
 
-We recommend using the `prometheusremotewrite` exporter when possible because the remote write ingest path is tested and proven at scale.
+Use the `prometheusremotewrite` exporter when possible because the remote write ingest path is tested and proven at scale.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/configure/configure-otel-collector.md:12]:
```suggestion
The remote write ingest path is tested and proven at scale. Therefore when possible, it is most performant to use the `prometheusremotewrite` exporter.
```

Comment (osg-grafana) [reply docs/sources/operators-guide/configure/configure-otel-collector.md:12]: Please vet for technical accuracy.

File: docs/sources/operators-guide/run-production-environment/planning-capacity.md [hunk 6]
```diff
@@ -138,7 +138,7 @@ When [remote]({{< relref "../architecture/components/ruler/index.md#internal" >}
 The [compactor]({{< relref "../architecture/components/compactor/index.md" >}}) component resources utilization is determined by the number of active series.
 
 The compactor can scale horizontally both in Grafana Mimir clusters with one tenant and multiple tenants.
-We recommend to run at least one compactor instance every 20 million active series ingested in total in the Grafana Mimir cluster, calculated before ingesters replication.
+Run at least one compactor instance every 20 million active series ingested in total in the Grafana Mimir cluster, calculated before ingesters replication.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/run-production-environment/planning-capacity.md:141]:
```suggestion
As a result, you can achieve high performance if you run at least one compactor instance every 20 million active series ingested in total in the Grafana Mimir cluster, calculated before ingesters replication.
```
This sentence is hard to parse. How can we break it up into smaller pieces.

File: docs/sources/operators-guide/run-production-environment/production-tips/index.md [hunk 7]
```diff
@@ -19,7 +19,7 @@ The ingester receives samples from distributor, and appends the received samples
 The per-tenant TSDB is composed of several files and the ingester keeps a file descriptor open for each TSDB file.
 The total number of file descriptors, used to load TSDB files, linearly increases with the number of tenants in the Grafana Mimir cluster and the configured `-blocks-storage.tsdb.retention-period`.
 
-We recommend fine-tuning the following settings to avoid reaching the maximum number of open file descriptors:
+To avoid reaching the maximum number of open file descriptors, fine tune the following settings:
```
Comment (osg-grafana) [inline docs/sources/operators-guide/run-production-environment/production-tips/index.md:22]: Nice. :)

File: docs/sources/operators-guide/run-production-environment/production-tips/index.md [hunk 8]
```diff
@@ -44,9 +44,8 @@ For these reasons, run the ingesters on disks such as SSDs that have fast disk s
 
 ### Ensure caching is enabled
 
-The querier supports caching to reduce the number API calls to the long-term storage.
+The querier supports caching. Ro reduce the number API calls to the long-term storage, enable caching in the querier.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/run-production-environment/production-tips/index.md:47]:
```suggestion
The querier supports caching. To reduce the number API calls to the long-term storage, enable caching in the querier.
```
Nice, even with with the typo. :)

File: docs/sources/operators-guide/run-production-environment/production-tips/index.md [hunk 9]
```diff
@@ -93,7 +91,7 @@ The store-gateway stores each block’s index-header on the local disk and loads
 The store-gateway keeps a file descriptor open for each index-header loaded at a given time.
 The total number of file descriptors used to load index-headers linearly increases with the number of blocks owned by the store-gateway instance.
 
-We recommend configuring the system's `file-max` ulimit at least to `65536` to avoid reaching the maximum number of open file descriptors.
+Configure the system's `file-max` ulimit at least to `65536` to avoid reaching the maximum number of open file descriptors.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/run-production-environment/production-tips/index.md:94]:
```suggestion
To avoid reaching the maximum number of open file descriptors, configure the system's `file-max` ulimit to at least `65536`.
```

Comment (osg-grafana) [reply docs/sources/operators-guide/run-production-environment/production-tips/index.md:94]: Rationale (I think, as a native speaker who sometimes does not know why) for changing the word order: "at least" is more tightly coupled / collocated with the value `65536` than it is to "ulimit". If any grammar nerds want to pipe up, I’d be curious why I changed this. :)

File: docs/sources/operators-guide/run-production-environment/production-tips/index.md [hunk 10]
```diff
@@ -113,12 +111,11 @@ For more information about required disk space, refer to [Compactor disk utiliza
 
 ### Ensure Memcached is properly scaled
 
-We recommend ensuring Memcached evictions happen infrequently.
+Ensure Memcached evictions happen infrequently.
 Grafana Mimir query performances might be negatively affected if your Memcached cluster evicts items frequently.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/run-production-environment/production-tips/index.md:115]:
```suggestion
Grafana Mimir query performances might be negatively affected if your Memcached cluster evicts items frequently.
To prevent this, make sure that Memcached evictions happen infrequently.
```

File: docs/sources/operators-guide/run-production-environment/production-tips/index.md [hunk 11]
```diff
@@ -113,12 +111,11 @@ For more information about required disk space, refer to [Compactor disk utiliza
...
-We recommend increasing your Memcached cluster replicas to add more memory to the cluster and reduce evictions.
+Increase your Memcached cluster replicas to add more memory to the cluster and reduce evictions.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/run-production-environment/production-tips/index.md:116]:
```suggestion
To add more memory to the cluster and reduce evictions, increase your Memcached cluster replicas.
```

File: docs/sources/operators-guide/run-production-environment/production-tips/index.md [hunk 12]
```diff
@@ -113,12 +111,11 @@ For more information about required disk space, refer to [Compactor disk utiliza
...
 
-We also recommend running a dedicated Memcached cluster for each type of cache: query results, metadata, index, and chunks.
-Running a dedicated Memcached cluster for each cache type is not required, but recommended so that each cache is isolated from the others.
+We also recommend running a dedicated Memcached cluster for each type of cache: query results, metadata, index, and chunks. While not required, it ensures that each cache is isolated from the others.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/run-production-environment/production-tips/index.md:118]:
```suggestion
To make sure that each cache is isolated from the another, (optionally) run a dedicated Memcached cluster for each type of cache: query results, metadata, index, and chunks.
```

File: docs/sources/operators-guide/secure/securing-alertmanager.md [hunk 13]
```diff
@@ -15,7 +15,7 @@ For example, the `webhook` receiver enables a user to configure an arbitrary URL
 If the Alertmanager network security is not hardened, Grafana Mimir users might configure the Alertmanager to issue requests to any network address both in the local network and the Internet.
 
 We recommend hardening the network on which the Alertmanager runs.
-Although hardening the network is out of the scope for Grafana Mimir, Grafana Mimir provides a basic built-in firewall that blocks connections created by Alertmanager receivers:
+Although hardening the network is out of the scope for Grafana Mimir, we reommend doing so where Alertmanager is running. Grafana Mimir provides a basic built-in firewall that blocks connections created by Alertmanager receivers:
```
Comment (osg-grafana) [inline docs/sources/operators-guide/secure/securing-alertmanager.md:18]:
```suggestion
Although hardening the network is out of scope for Grafana Mimir, we recommend doing so where Alertmanager is running. Grafana Mimir provides a basic built-in firewall that blocks connections created by Alertmanager receivers:
```

Comment (osg-grafana) [reply docs/sources/operators-guide/secure/securing-alertmanager.md:18]: Probably can leave this recommend phrase as is.

**Review by osg-grafana — APPROVED** [review]
Unblocking with several pieces of feedback

**Comment by osg-grafana** [comment]
@pracucci, can you add some rationale as to why there are so many recommendations? I think a little rationale can go a long way.

**Comment by konstantinos-null** [comment]
Hey @osg-grafana, first of all thank you for all the valuable feedback, it's nice getting a bit more insight on the project. I'll go over the changes in the next couple of days and push some change for either approval or more feedback!

**Comment by replay** [comment]
The CHANGELOG has just been cut to prepare for the next Mimir release. Please rebase `main` and eventually move the CHANGELOG entry added / updated in this PR to the top of the CHANGELOG document. Thanks!

**Comment by osg-grafana** [comment]
@konstantinos-null, @pracucci and I will incorporate the style changes in subsequent PRs are those files are touched. Thank you for providing feedback that we can refer back to.
